mreg_cli/outputmanager.py

```diff
@@ -405,9 +404,9 @@ def get_filter(self, command: str) -> tuple[str, Any, bool]:
 
     def add_formatted_table(
         self,
-        headers: Sequence[str],
-        keys: Sequence[str],
-        data: Sequence[dict[str, Any] | BaseModel],
+        headers: Iterable[str],
+        keys: Iterable[str],
+        data: Iterable[dict[str, Any] | BaseModel],
```

Comment:
Iterable in case we have some sort of comprehension or generator that we pass in. We don't need access to indexing. We can remove this, I don't feel too strongly about it, but one could imagine passing in a generator expression or similar to `output_multiple`, which does not fulfill the `Sequence` interface.

Reply:
Iterable is a good solution. :)